background/score/activities.go

```diff
@@ -162,6 +163,12 @@ func (s *ScoreUpdateWorker) RefreshLocationStateActivity(ctx context.Context, ac
 			return nil, err
 		}
 
+		if time.Since(profile.LastNudge[schema.BehaviorOnSymptomSpikeNudge]) > 90*time.Minute { // 90 minutes of delay between nudges
+			if profile.Metric.SymptomDelta <= 110 && metric.SymptomDelta > 110 { // from a non-spike area to a spike area
```

Comment:
``
```suggestion
			if profile.Metric.SymptomDelta <= 10 && metric.SymptomDelta > 10 { // from a non-spike area to a spike area
```
Base on the calculation of delta